### PR TITLE
[IMP] website_profile: Convert all jquery code in vanilla Javascript

### PR DESCRIPTION
**Pull request by msh-odoo**
Description of the issue/feature this PR addresses:

This PR aim to convert all jQuery code into Vanilla JS in website_profile, this way we will reduce the dependency of jQuery in Odoo codebase.

task-3770362

### CODE REVIEW

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by divy-odoo** [comment]
Hello @bvr-odoo, Rebase done. :+1:  PR is ready for review now.

Thanks : )

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
Hello, small changes to do.
Also your PR title and commit title are different
PR => `[REF] Convert all jquery related code in vanilla Javascript`
Commit => `[IMP] Convert all jquery related code in vanilla Javascript`

+ Missing module name in commit title ...

**Comment by divy-odoo** [comment]
Hello @bvr-odoo , Applied the suggested changes.
thanks : )

File: addons/website_profile/static/src/js/website_profile.js [hunk 1]
```diff
@@ -57,23 +58,23 @@ publicWidget.registry.websiteProfileEditor = publicWidget.Widget.extend({
             return def;
         }
 
-        const $textarea = this.$("textarea.o_wysiwyg_loader");
+        const textareaEl = document.querySelector("textarea.o_wysiwyg_loader");
```
Comment (bvr-odoo) [inline addons/website_profile/static/src/js/website_profile.js:61]: `this.el` should also works (same remark as [here](https://github.com/odoo/odoo/pull/165087#discussion_r1705256032)) ... If it's true change it please

**Review by bvr-odoo — CHANGES_REQUESTED** [review]

File: addons/website_profile/static/src/js/website_profile.js (hunk 1 above)
Comment (divy-odoo) [reply addons/website_profile/static/src/js/website_profile.js]: Yes, You are right!! :slightly_smiling_face:

**Comment by divy-odoo** [comment]
Hello @bvr-odoo, Adapted the chanegs. Thanks : )

**Review by bvr-odoo — APPROVED** [review]

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**